mock/mock_test.go

```diff
@@ -1271,7 +1271,7 @@ func Test_Arguments_Diff_WithArgMatcher(t *testing.T) {
 	assert.Equal(t, 1, count)
 	assert.Contains(t, diff, `(bool=false) not matched by func(int) bool`)
 
-	diff, count = args.Diff([]interface{}{"string", 123, false})
+	diff, _ = args.Diff([]interface{}{"string", 123, false})
```

Comment:
Shouldn't we check the value of `count` instead of ignoring it?

Reply:
You're right, will fix!